pydicom/pixel_data_handlers/pillow_handler.py

```diff
@@ -80,6 +80,33 @@ def should_change_PhotometricInterpretation_to_RGB(dicom_dataset):
     return False
 
 
+def _decompress_single_frame(data, transfer_syntax, photometric_interpretation):
+    fio = io.BytesIO(data)
```

Comment:
Can you add a docstring please?

Reply:
Addressed with 7f081202a48caad0c2b8c5061e95f034e4b6d0ed